### PR TITLE
docs: improve the documentation of alpine

### LINKED ISSUE #132: request help：failed to build image from source
1、clone source

```shell
git clone https://github.com/apache/apisix-docker.git
```

2、Build Image

build alpine failed：

```shell
$ cd apisix-docker
$ export APISIX_VERSION=2.3
$ docker build -t apisix:${APISIX_VERSION}-alpine --build-arg APISIX_VERSION=${APISIX_VERSION} -f alpine/Dockerfile alpine
Sending build context to Docker daemon 6.656 kB
Step 1/17 : ARG ENABLE_PROXY=false
Please provide a source image with `from` prior to commit
```
I don't know what caused this problem.

### PR DESCRIPTION
**Pull request by Firstsawyou**
close #132

### CODE REVIEW

File: alpine/README.MD [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ### When building with `alpine` basic image, please specify `APISIX_VERSION` through `build-arg`
 
 ```
-docker build --build-arg APISIX_VERSION=$APISIX_VERSION -t $IMAGE_NAME .
-```
\ No newline at end of file
+$ docker build --build-arg APISIX_VERSION=$APISIX_VERSION -t $IMAGE_NAME .
+```
+
+Note: The minimum version of docker that supports building image is `docker 17.05.0-ce`.
```
Comment (Firstsawyou) [inline alpine/README.MD:7]: @gxthrj  @tokers  I am not sure about the minimum version of docker. So I temporarily wrote the docker 17.05.0-ce version.
What should be the minimum version of docker?

File: alpine/README.MD [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ### When building with `alpine` basic image, please specify `APISIX_VERSION` through `build-arg`
 
 ```
```
Comment (tokers) [inline alpine/README.MD:3]: Add language specifier `shell`.

File: alpine/README.MD (hunk 1 above)
Comment (tokers) [reply alpine/README.MD:7]: Just add a warning message is enough such as "if you occur errors like blah blah blah, please upgrade your docker version".

Comment (gxthrj) [reply alpine/README.MD:7]: I think use `docker 17.05.0-ce` is ok

Ref to: 
https://github.com/docker/docker/pull/31352
https://docs.docker.com/engine/release-notes/17.05

**Review by gxthrj — APPROVED** [review]
Great

File: alpine/README.MD (hunk 2 above)
Comment (Firstsawyou) [reply alpine/README.MD]: updated.

**Review by membphis — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Pull request merged**